### ISSUE TITLE
Table columns structure in ZooKeeper is different from local table structure

### ISSUE THREAD

**Issue by cw9** [ClickHouse/ClickHouse#10074]
I'm getting this error 
`Code: 122, e.displayText() = DB::Exception: Table columns structure in ZooKeeper is different from local table structure (version 20.3.5.21 (official build))`

This can be reproduced by creating the first replica of table1 with column a,b,c and then creating the second replica with columns a,c,b. 

It seems that the ZK already knows what the column structure of table1 should be, is there a way to create the second table from that structure on ZK? So that we can avoid providing the wrong column structure of a,c,b?

**Comment by alexey-milovidov** [comment]
Yes, it's a great feature request. We need to allow creating replicas without specifying the schema.
